nx-X11/programs/Xserver/xkb/xkb.c

```diff
@@ -6200,7 +6200,7 @@ XkbResetProc(ExtensionEntry *extEntry)
 }
 
 void
-XkbExtensionInit(INITARGS)
```

Comment:
> INITARGS remains defined, and used in xkb and other places in the code. Seems inconsistent 
> to remove it only from one place. 

Good point. I checked what X.org did with that and found (and backported the below commit on X.org's xserver Git repo:

```
commit 5079db78aea8fa42e781dd876c1ee1c31571b3ae
Author: Tomas Carnecky <tom@dbservice.com>
Date:   Tue Jul 10 02:02:53 2012 +0100

    Replace INITARGS with void
    
    INITARGS was a hardcoded define to void.  Since knowing the function
    signature for your extensions is kinda useful, just replace it with a
    hardcoded void, but leave the define there for API compatibility.
    
    Signed-off-by: Tomas Carnecky <tom@dbservice.com>
    Reviewed-by: Daniel Stone <daniel@fooishbar.org>
    Reviewed-by: Cyril Brulebois <kibi@debian.org>
    Reviewed-by: Jamey Sharp <jamey@minilop.net>
    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
    Signed-off-by: Keith Packard <keithp@keithp.com>
```